scipy/special/_support_alternative_backends.py

```diff
@@ -51,22 +55,38 @@ def wrapped(*args, **kwargs):
 
 
 array_special_func_map = {
+    'logit': 1,
+    'expit': 1,
     'log_ndtr': 1,
     'ndtr': 1,
     'ndtri': 1,
+    'digamma': 1,
+    'polygamma': 2,
+    'multigammaln': 2,
+    'gammaln': 1,
+    'gamma': 1,
+    'gammainc': 2,
+    'gammaincc': 2,
+    'betaln': 2,
+    'beta': 2,
+    'betainc': 3,
     'erf': 1,
     'erfc': 1,
+    'erfinv': 1,
+    'zeta': 2,
+    # No one implements binom yet.
+    # 'binom': 2,
```

Comment:
```suggestion
    'binom': 2,
```
Dask and MArray implement _all elementwise ufuncs_ when they wrap around numpy!

Reply:
Great!

If we expose `is_dask_namespace`, I can test for that and let `binom` through for that case.

How do I test for MArray?

Reply:
> Great!
> 
> If we expose `is_dask_namespace`, I can test for that and let `binom` through for that case.

Please merge from main, there's news for you :wink: #22639

> How do I test for MArray?

At the moment, specific special functions that are needed by scipy.stats are indirected tested in there.
For a more systematic test effort, I would advise to talk it through thoroughly with @mdhaber and write a dedicated PR for it.

Reply:
`binom` is not a function; it's a namespace for distributions (`binoms.ppf` etc.). This delegation system is not fit for purpose for it.

Reply:
Sorry, I'm confused here.  `scipy.stats.binom` is a function:
```python
[utM] In [10]: ss.binom(5, 2)
Out[10]: np.float64(10.0)
```

Reply:
Apologies, I got confused with `scipy.stats.binom`